### PR TITLE
Completes REPMFFORM-142 and 143

### PR DESCRIPTION
**Pull request by BAucoinHC**

### CODE REVIEW

File: projects/master-file/src/app/regulatory-information/regulatory-information.service.ts
Comment (linghanHC) [inline projects/master-file/src/app/regulatory-information/regulatory-information.service.ts:1]: @BAucoinHC see my comments for https://jill.hc-sc.gc.ca/jira/browse/REPMFFORM-143. both of En and Fr need to be saved for transaction_description. This needs to be consistent across all apps.

File: projects/master-file/src/assets/data/txDescriptions.json
Comment (linghanHC) [inline projects/master-file/src/assets/data/txDescriptions.json:1]: this value can be changed to 13 or you can advise clients to update it to 21 in their matrix.

File: projects/master-file/src/app/regulatory-information/regulatory-information.service.ts
Comment (linghanHC) [reply projects/master-file/src/app/regulatory-information/regulatory-information.service.ts:1]: did you checkin all your codes? did you change the Enrollment model and regulatory-information.html?

Comment (BAucoinHC) [reply projects/master-file/src/app/regulatory-information/regulatory-information.service.ts:1]: I think so, regulatory-information.html I definitely changed, the changes should be there.

File: projects/master-file/src/assets/data/txDescriptions.json
Comment (BAucoinHC) [reply projects/master-file/src/assets/data/txDescriptions.json:1]: I have changed the value to 13 as needed in the code.

**Review by AMckayHC — APPROVED** [review]

**Pull request merged**